ods_ci/tests/Resources/OCP.resource

```diff
@@ -338,6 +338,33 @@ Get Resource Attribute
     END
     RETURN    ${value}
 
+Wait For Installplan And Approve It
+    [Documentation]    Wait for the operator's installplan to appear then approve it in case of Manual installplan approval
+    [Arguments]    ${namespace}    ${operator_name}     ${subscription_name}    ${operator_version}=${EMPTY}
+    ${csv} =    Evaluate    '${operator_name}.${operator_version}' if '${operator_version}' else '${operator_name}'
+    Log    The csv is: ${csv}    console=yes
+    Wait Until Keyword Succeeds    10m    5s    Check If Resource Exists By Command Output    oc get installplans -n ${namespace} -o yaml | grep "${csv}"
```

Comment:
maybe i'd rather create a keyword like `Search Install Plan` implemented like

```
${rc} ${out}=        Run and Return Rc and Output      oc get installplan -ojson | jq '.items[].spec.clusterServiceVersionNames[] | select(. | contains("${csv}"))'
Should Not Be Empty    ${out}
```

`Check If Resource Exists By Command` sounds yet another variant of existence check:  `Resource Should Exist`, `Is Resource Present`, `Resource Status Should Be` .

Reply:
updated